sdk/src/main/res/values-th/strings.xml

```diff
@@ -107,4 +108,15 @@
     <string name="payment_method_mobile_banking_bay_title">KMA</string>
     <string name="payment_method_mobile_banking_bbl_title">Bualuang mBanking</string>
     <string name="googlepay">Google Pay</string>
-</resources>
\ No newline at end of file
+    <string name="payment_method_atome_title">Atome</string>
+    <string name="label_name">ชื่อ (ไม่จำเป็น)</string>
+    <string name="label_email_optional">อีเมล (ไม่จำเป็น)</string>
+    <string name="label_shipping">ที่อยู่ในการจัดส่ง</string>
+    <string name="label_billing">ที่อยู่สำหรับส่งใบแจ้งหนี้ (ไม่จำเป็น)</string>
+    <string name="label_street">ที่อยู่</string>
+    <string name="label_postal">รหัสไปรษณีย์</string>
+    <string name="label_city">จังหวัด</string>
+    <string name="label_country_code">รหัสประเทศ</string>
+    <string name="label_billing_shipping">ที่อยู่เดียวกันทั้งจัดส่งและแจ้งหนี้</string>
```

Comment:
I think it's a bit informal. May be something like this.
```suggestion
    <string name="label_billing_shipping">ใช้ที่อยู่เดียวกันสำหรับจัดส่งสินค้าและใบแจ้งหนี้</string>
```

Reply:
Sure 😂